### PR TITLE
for issue #192 added hyphen

### PR DESCRIPTION
**Pull request by ramvikrams**
## Description
- changed files for hyphen instead of underscore and changed the respective tests for it

## Affected Issues
- #192

### CODE REVIEW

File: src/opera/pge/base/base_pge.py [hunk 1]
```diff
@@ -346,7 +346,7 @@ def _core_filename(self, inter_filename=None):  # pylint: disable=unused-argumen
         """
         time_tag = get_time_for_filename(self.production_datetime)
 
-        return f"{self.PROJECT}_{self.LEVEL}_{self.NAME}_{time_tag}"
+        return f"{self.PROJECT}_{self.LEVEL}_{self.NAME}-{time_tag}"
```
Comment (collinss-jpl) [inline src/opera/pge/base/base_pge.py:349]: Hi @ramvikrams, thanks for putting this PR together. My only note is that the underscore to hyphen change does not need to be applied to the placeholder filename we apply in `base_pge.py`. If you could revert this change I would be happy to merge the rest of this branch.

**Review by collinss-jpl — CHANGES_REQUESTED** [review]

File: src/opera/pge/base/base_pge.py (hunk 1 above)
Comment (ramvikrams) [reply src/opera/pge/base/base_pge.py]: Done

**Review by collinss-jpl — APPROVED** [review]
Looks good, thanks again @ramvikrams for pitching in!
